tests/utils/SmoothTest.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2023 UltiMaker
+// CuraEngine is released under the terms of the AGPLv3 or higher
+
+#include <gtest/gtest.h>
+
+#include <set>
+
+#include <range/v3/all.hpp>
+
+#include "utils/IntPoint.h"
+#include "utils/actions/smooth.h"
+#include "utils/polygon.h"
+
+namespace cura
+{
+
+
+TEST(SmoothTest, TestSmooth)
+{
```

Comment:
Needs a bit of documentation; Why is this a good test? What is it trying to do?  (nitpicking)

Reply:
Test was used during debugging, for quick iterative cycles. I believe it also fails currently. In will create a couple actual useful test cases, tomorrow